### DISCUSSION TITLE
Are there any functional (even if minor) differences between the Chrome and Firefox versions?

### DISCUSSION THREAD

**Discussion by hrxn** [IsThereAnyDeal/AugmentedSteam#1807] [Q&A] [answered]
(see title)

**Comment by candela97** [comment, marked as answer]
No, all functionality is the same.

**Comment by J-SoG-YK** [comment]
Sorry to necro this topic.  But I have noticed that there is one functional difference. With Chromium browsers, you can pin the Augmented Steam button to the toolbar. It is not possible to do so on Mozilla based browsers. Not that it's really that big a deal. Most of the functionality is duplicated in the Context Menu, anyway. I really only bring this up because I am curious as to why?
